docs/en/core/editor.md

```diff
@@ -25,7 +25,7 @@ Editing text with Zettlr is simple. Just create a new file or select an existing
 
 * `Cmd/Ctrl+B`: Make text **bold**
 * `Cmd/Ctrl+I`: _Emphasize_ text
-* `Cmd/Ctrl+K`: Create a new `[link](https://www.zettlr.com)`. Selected text will become the description—you only need to add the URL. And if there is a valid URL in the clipboard, Zettlr even takes it as a URL, meaning you don't have to do anything on your own!
+* `Cmd/Ctrl+K`: Create a new `[link](https://www.zettlr.com)`. Selected text will become the description —you only need to add the URL. And if there is a valid URL in the clipboard, Zettlr even takes it as a URL, meaning you don't have to do anything on your own!
```

Comment:
There's either a space too much or a space too less (depending on which style you prefer)

Reply:
BTW: I sent a request to join the forum, but did not receive an approval. Is it because of the move or did the moderator maybe not see the request?

Reply:
>  Is it because of the move or did the moderator maybe not see the request?

Ah, maybe the forum is the cause why I'm receiving numerous error messages from the Mailer Daemon 😅 I was thinking about that all day :D I'll have a look!

Reply:
So the emails should work, just tested it, if you tell me your username I can manually approve you

Reply:
I received an email but the fourm says:

> We switched to a manual approval system for registration on this forum, as this makes it easier to keep bots out of the forum. We'll approve everybody who doesn't look like a bot!

My username is canpolat

Reply:
> We switched to a manual approval system for registration on this forum, as this makes it easier to keep bots out of the forum. We'll approve everybody who doesn't look like a bot!

That's actually a lie, I switched back to automatic approval, because I'm lazy, I should remove that message, yup!

Reply:
I see... Then I suppose my account is in some sort of limbo. I can log in, but I don't see any options to create a post or reply to a conversation.